packages/web-components/fast-foundation/src/accordion-item/accordion-item.ts

```diff
@@ -22,6 +22,22 @@ export type AccordionItemOptions = FoundationElementDefinition &
 
 /**
  * An individual item in an {@link @microsoft/fast-foundation#(Accordion:class) }.
+ *
+ * @slot start - Content which can be provided between the heading and the icon
+ * @slot end - Content which can be provided between the start slot and icon
+ * @slot heading - Content which serves as the accordion item heading and text of the expand button
+ * @slot - The default slot for accordion item content
+ * @slot - expanded-icon - The expanded icon
```

Comment:
The analyzer uses the hyphen as the separator between the name and the description. These hyphens between @slot and the name should be removed.

Reply:
Yeah, I did almost all of these manually, I'll do one last check for any time it's not the default slot.